cypress/support/pages/news/pageCommonNews.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+class News_Common {
+
+  static #newsOverviewNavigationButton = '[data-testid="Neuigkeiten"]'
```

Comment:
Why is this data-testid in german? The other ones are in english?

We should use the same language. And especially ensure that it is not depending from the chosen language.